Bottom: e8124256053121f4c35d0f163125702b7dea4c57
Top:    3df0369a72987bf3ee81dfddd55c18c542bcf024
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2017-03-17 23:18:06 +0000

Refresh of remove-unplacedtype-and-just

---

diff --git a/src/action.c b/src/action.c
index f7fb709..5c05c1d 100644
--- a/src/action.c
+++ b/src/action.c
@@ -7157,8 +7157,12 @@ ActionElementList (int argc, char **argv, Coord x, Coord y)
       END_LOOP;
       if (number_of_footprints_not_found > 0)
 	gui->confirm_dialog (_("Not all requested footprints were found.\n"
-			     "See the message log for details"),
+			     "See the message log for details.\n"
+                             "Place elements from the library window \"unplaced\" tab"),
 			     "Ok", NULL);
+      else
+        gui->confirm_dialog (_("Place elements from the library window \"unplaced\" tab"),
+                             "Ok", NULL);
       return 0;
     }
